profile/README.md

```diff
@@ -1,48 +1,40 @@
-# The Camunda Community Hub
+# Welcome to the Camunda Community Hub!
+## A project to empower our open source community extension maintainers, and encourage new contributors to get started contributing to the Camunda open source ecosystem. 
 
-The Camunda Community Hub is Camunda's GitHub Organization for community contributed extensions. Those extensions are independently maintained by members of the Camunda open source community (with some maintainers happen to be Camunda employees too).
+Extensions in the Camunda Community Hub organization are independently led and maintained projects by members of the broader Camunda open source community, some of whom happen to be Camunda employees, or are projects/tutorials that are created and maintained by the Camunda Developer Relations team!
 
-With the Camunda Community Hub we want to empower our open source community to contribute, and encourage new contributors to get started contributing to the Camunda open source ecosystem.
+With the Camunda Community Hub, we want to empower our open source community to contribute, and encourage new contributors to get started contributing to the Camunda open source ecosystem.
 
-## Existing community extensions
-
-You can find a list of existing community extensions by [browsing the repostories](https://github.com/orgs/camunda-community-hub/repositories) in this GitHub organization.
+**Table of Contents**
 
-## FAQ and further information
+* [Existing community extensions](#existing-community-extensions)
+* [How to contribute](#how-to-contribute)
+* [Troubleshooting](#troubleshooting)
 
-We’ve put together some commonly asked questions and answers on https://github.com/camunda-community-hub/community. If you have further questions, you can also ask via our [forum](https://forum.camunda.io/) or reach out to the [Camunda DevRel team](https://github.com/orgs/camunda-community-hub/teams/devrel) via email: community@camunda.com.
+## Existing community extensions
 
-## How to get my community extension to the Camunda Community Hub?
+You can find a list of existing community extensions by [browsing the repostories](https://github.com/orgs/camunda-community-hub/repositories) in this GitHub organization.
 
-- [Open an issue](https://github.com/Camunda-Community-Hub/community/issues/new/) in the [community repository](https://github.com/Camunda-Community-Hub/community/) and ask to be invited to join the Camunda Community Hub (if you are not already a collaborator).
+You can also browse by Awesome Lists created by the community.
+- [Awesome Camunda External Task Clients](https://github.com/camunda-community-hub/awesome-camunda-external-clients)
+- [Awesome Camunda Platform 8 Projects](https://github.com/camunda-community-hub/awesome-camunda-platform-8)
+- [Awesome Camunda and Robot Framework Projects](https://github.com/camunda-community-hub/awesome-Camunda-and-Robotframework-projects)
```

Comment:
You may want to make it a little clearer about the 2 Awesome lists that are for C7. We made some choices during the C8 launch to be pretty overt about this - explicitly name things for C7 and C8. This is a recommendation, but you could easily add something to describe each of these Awesome lists directly in the list.